db/migrate/20220722133738_add_delete_caselogs_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddDeleteCaselogsUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_foreign_key :case_logs, :organisations, column: "owning_organisation_id", on_delete: :cascade
```

Comment:
Do we not need to remove these first? Did they not exist in the first place?

Reply:
This relationship doesn't appear in the schema in main so I don't think it existed previously.